gensim/models/word2vec.py

```diff
@@ -1181,33 +1189,103 @@ def intersect_word2vec_format(self, fname, lockf=0.0, binary=False, encoding='ut
         logger.info("merged %d vectors into %s matrix from %s" % (overlap_count, self.wv.syn0.shape, fname))
 
     def most_similar(self, positive=[], negative=[], topn=10, restrict_vocab=None, indexer=None):
+        """
+        Please refer to the documentation for
+        `gensim.models.KeyedVectors.most_similar`
+        This is just a forwarding function.
+        In the future please use the `gensim.models.KeyedVectors` instance in wv
```

Comment:
Strange line breaks + docstring a bit obscure. I'm not sure reading this would help me if I didn't already know what it's talking about.

How about simply `Deprecated; use self.ww.most_similar() instead.`?

Same with all the other "forwarding" functions.